fellowship_testimonials.module

```diff
@@ -86,10 +86,11 @@
  <source src={$row->testimonial_video} type='video/mp4'>
                                 </video> ";
  while($speakerrow = $speakerresult->fetchObject()){
-        $page_content .= "<p><span>{$speakerrow->name}</span><br>Institute: {$speakerrow->institute}, {$speakerrow->place}<br><br><span style='margin-right:0;'>";
+        $page_content .= "<p><span>{$speakerrow->name}</span><br>Institute: {$speakerrow->institute}, {$speakerrow->place}<br><br><span style='margin-right:0; size: 10px;'>";
     }
-            $page_content .= "Fellowship task</span>: {$row->fellowship_task}
-            <br><br><span>Fellowship Year</span>: {$row->year}</p></div>";
+            $page_content .= "Activity name</span>: {$row->activity}
```

Comment:
Just display the activity name in a large font without the title 'Activity name"